internal/pkg/config/runtime.go

```diff
@@ -5,7 +5,8 @@
 package config
 
 type Runtime struct {
-	GCPercent int `config:"gc_percent"`
+	GCPercent   int   `config:"gc_percent"`
+	MemoryLimit int64 `config:"memory_limit"`
```

Comment:
Should we add a comment stating that this is set to `math.MaxInt64` by default; and setting this too low will result in a lot more GC behaviour (as outlined in the docs)?

Reply:
updated